src/store/isConfigured.js

```diff
@@ -2,7 +2,7 @@ import { applyReducerHash } from '@redhat-cloud-services/frontend-components-uti
 
 const initialState = {
     loading: false,
-    serverError: {},
+    serverError: '',
```

Comment:
@r14chandra can you please share the reason on why you changed it from object to a string? I normally prefer to store entire error object and use the information I want for error validation.

Reply:
How do you use this object to check the error. Could you point me where we have used anywhere in our codebase?
Though I can add the check for length of the object but just want see how you use it.